thinc/util.py

```diff
@@ -47,10 +47,12 @@ def get_torch_default_device() -> "torch.device":
 
 
 def get_array_module(arr):  # pragma: no cover
-    if is_cupy_array(arr):
+    if is_numpy_array(arr):
+        return numpy
+    elif is_cupy_array(arr):
         return cupy
     else:
-        return numpy
+        return None
```

Comment:
I'd say that it would be more appropriate to raise a `ValueError` here, given (my reading of) the function's semantics (`arr` is implicitly expected to be an array type, so any other input is inherently invalid in the context of the function).